### ISSUE TITLE
HttpError: specifying shard-group duration is not supported when using InfluxDB Cloud

### ISSUE THREAD

**Issue by BenS89** [ioBroker/ioBroker.influxdb#313]
Das Problem scheint schon älter zu sein: https://forum.iobroker.net/topic/46098/test-adapter-influxdb-2-0/151?lang=de
Bei Verwendung der InfluxDB Cloud kommt permanent folgender Fehler im Log: 

![image](https://user-images.githubusercontent.com/5046137/206730579-05ebe661-983d-45ed-b4b9-e3a4cce79ef9.png)

Gibt es hierzu einen Fix oder Workaround?

**Comment by BenS89** [comment]
Wenn der Haken "Pings zur Verfügbarkeitsprüfung an die Datenbank senden" in den Experteneinstellungen gesetzt ist (müsste per default meine ich so sein), kommt die Meldung alle paar Sekunden und der Adapter startet neu. Erst wenn ich den Haken weg nehme kommt der Fehler einmalig beim Start des Adapters

**Comment by matzeapps** [comment]
Gibt es hierzu schon etwas Neues?

**Comment by Apollon77** [comment]
Nein, noch keine Zeit gehabt. Ebenso würde ein Debug log vllt auch helfen zu verstehen wann die Meldung kommt

**Comment by Marc-Berg** [comment]
> Ebenso würde ein Debug log vllt auch helfen zu verstehen wann die Meldung kommt

Bei Start des Adapters (oder beim "Ping") wird die in der Instanz eingestellte Retentiontime jedes Mal in die DB geschrieben, das ist in der Cloud nicht erlaubt. Wenn ich den [Quellcode ](https://github.com/ioBroker/ioBroker.influxdb/blob/234de8d73a8e1b48aabce21d9133eb8a13af2956/lib/DatabaseInfluxDB1x.js#L82)richtig verstehe, dürfte dieses Überschreiben nicht passieren, wenn die in der DB/Cloud eingestellte mit der in der Instanz hinterlegeten Retentiontime übereinstimmt.
